PRODUCTION.md

```diff
@@ -0,0 +1,82 @@
+# Using S3 for continuous integration testing with Docker
+
+When you start the Scality S3server image, you can adjust the configuration of
+the Scality S3server instance by passing one or more environment variables
+on the docker run command line.
+
+## Environment Variables
+
+### HOST_NAME
+
+This variable specifies an host name:
```

Comment:
@rahulreddy - thanks for the recommendation.
Just to be sure, we need to specify hostname for every requests (hosted/path style): https://github.com/scality/S3/blob/4236068c5dc3863362a47a0d3af7c86dd1359fd0/lib/utils.js#L34 . Is it correct?